autosklearn/automl.py

```diff
@@ -917,6 +936,168 @@ def refit(self, X, y):
+
+        exclude = dict()
+        include = dict()
+        if self._include_preprocessors is not None and self._exclude_preprocessors is not None:
+            raise ValueError('Cannot specify include_preprocessors and '
+                             'exclude_preprocessors.')
+        elif self._include_preprocessors is not None:
+            include['feature_preprocessor'] = self._include_preprocessors
+        elif self._exclude_preprocessors is not None:
+            exclude['feature_preprocessor'] = self._exclude_preprocessors
+
+        if self._include_estimators is not None and self._exclude_estimators is not None:
+            raise ValueError('Cannot specify include_estimators and '
+                             'exclude_estimators.')
+        elif self._include_estimators is not None:
+            if task in CLASSIFICATION_TASKS:
+                include['classifier'] = self._include_estimators
+            elif task in REGRESSION_TASKS:
+                include['regressor'] = self._include_estimators
+            else:
+                raise ValueError(task)
+        elif self._exclude_estimators is not None:
+            if task in CLASSIFICATION_TASKS:
+                exclude['classifier'] = self._exclude_estimators
+            elif task in REGRESSION_TASKS:
+                exclude['regressor'] = self._exclude_estimators
+            else:
+                raise ValueError(task)
+
+        # Prepare missing components to the TAE function call
+        if 'include' not in kwargs:
+            kwargs['include'] = include
+        if 'exclude' not in kwargs:
+            kwargs['exclude'] = exclude
+        if 'memory_limit' not in kwargs:
+            kwargs['memory_limit'] = self._memory_limit
+        if 'resampling_strategy' not in kwargs:
+            kwargs['resampling_strategy'] = self._resampling_strategy
+        if 'metric' not in kwargs:
+            kwargs['metric'] = self._metric
+        if 'disable_file_output' not in kwargs:
+            kwargs['disable_file_output'] = self._disable_evaluator_output
+        if 'pynisher_context' not in kwargs:
+            kwargs['pynisher_context'] = self._multiprocessing_context
+        if 'stats' not in kwargs:
+            scenario_mock = unittest.mock.Mock()
+            scenario_mock.wallclock_limit = self._time_for_task
+            kwargs['stats'] = Stats(scenario_mock)
+        kwargs['stats'].start_timing()
+
+        # Allow to pass the cutoff also as an argument
+        cutoff = kwargs.pop('cutoff', self._per_run_time_limit)
+
+        # Fit a pipeline, which will be stored on disk
+        # which we can later load via the backend
+        ta = ExecuteTaFuncWithQueue(
+            backend=self._backend,
+            autosklearn_seed=self._seed,
+            abort_on_first_run_crash=False,
+            cost_for_crash=get_cost_of_crash(kwargs['metric']),
+            port=self._logger_port,
+            **kwargs,
+            **self._resampling_strategy_arguments
```

Comment:
Don't we need to also pass the resampling strategy?

Reply:
It is a couple of lines above, in the kwargs